### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by JeffCarver**
Still need to fix the email link, the issues link, and the lessons link.

### CODE REVIEW

**Comment by JeffCarver** [comment]
Fixes #12

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -126,26 +112,13 @@ your clone of the originating `gh-pages` branch.
...
-happens on the [discussion mailing list][discuss-list],
-which everyone is welcome to join.
-You can also [reach us by email][email].
-
-[email]: mailto:team@carpentries.org
-[dc-issues]: https://github.com/issues?q=user%3Adatacarpentry
-[dc-lessons]: http://datacarpentry.org/lessons/
-[dc-site]: http://datacarpentry.org/
-[discuss-list]: https://carpentries.topicbox.com/groups/discuss
+[email]:
```
Comment (cosden) [inline CONTRIBUTING.md]:
```suggestion
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -126,26 +112,13 @@ your clone of the originating `gh-pages` branch.
...
-
-[email]: mailto:team@carpentries.org
-[dc-issues]: https://github.com/issues?q=user%3Adatacarpentry
-[dc-lessons]: http://datacarpentry.org/lessons/
-[dc-site]: http://datacarpentry.org/
-[discuss-list]: https://carpentries.topicbox.com/groups/discuss
+[email]: 
+[i-site]: https://intersect-training.org/
+[i-issues]: tbd
+[i-lessons]: tbd
```
Comment (cosden) [inline CONTRIBUTING.md]:
```suggestion
```

Comment (cosden) [reply CONTRIBUTING.md]: I don't think we need these, and would rather just drop them than leave them here to confuse.

File: CONTRIBUTING.md (hunk 1 above)
Comment (JeffCarver) [reply CONTRIBUTING.md]: I agree. I will remove this.

File: CONTRIBUTING.md (hunk 2 above)
Comment (JeffCarver) [reply CONTRIBUTING.md]: Should we keep the "issues" one? We can point it to the issues in this repo. Or we can just delete them both.

Comment (cosden) [reply CONTRIBUTING.md]: It's not referenced/linked right now so let's remove it and add things back in for each lesson where needed.

**Pull request merged**